ChangeLog

```diff
@@ -1,6 +1,598 @@
 CHANGES
 =======
 
+* Fixed positive testcase for GetFlavorObject execute method
```

Comment:
You can remove this file

Reply:
This is add by Hunter, maybe we can check this with him later.

Reply:
ok

Reply:
Yeah let's remove it. These are auto added or something due to pbr tool